models/intermediate/trades/int_trade_agg_day.sql

```diff
@@ -25,8 +25,8 @@ with
             , buying_amount
         from {{ ref('stg_history_trades') }}
         where
-            ledger_closed_at < timestamp(date_add(date('{{ dbt_airflow_macros.ds() }}'), interval 1 day))
-            and ledger_closed_at >= timestamp(date('{{ dbt_airflow_macros.ds() }}'))
+            ledger_closed_at < timestamp(date_add(date('{{ dbt_airflow_macros.ts(timezone=none) }}'), interval 1 day))
```

Comment:
nit: I think you can use [timestamp_trunc()](https://cloud.google.com/bigquery/docs/reference/standard-sql/timestamp_functions#timestamp_trunc) instead of converting timestamp to date back to timestamp to get the day part of the `dbt_airflow_macros.ts`

I would assume the BQ query planner optimizes this away so there is no performance impact so it's purely a coding style suggestion

Reply:
Sounds good, will make change. I missed it in effort of making bulk changes

Reply:
Addressed in https://github.com/stellar/stellar-dbt-public/pull/79/commits/b55bff584146a58251fc68c413495df62b7559cb